pkg/server/v3/server.go

```diff
@@ -28,7 +28,6 @@ import (
 
 	clusterservice "github.com/envoyproxy/go-control-plane/envoy/service/cluster/v3"
 	discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
-	discoverygrpc "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
```

Comment:
Note: this is an artifact of google import. Google tooling puts gRPC and protos into separate packages.

Reply:
It seems all the builds passed, would it be worth keeping them separate for the sake of distinguishing between the two? Or since it seemed to build fine merge and let staticcheck be happy?

Reply:
If we can keep`discoverygrpc`  import and delete `discovery `?